geode-core/src/main/java/org/apache/geode/internal/serialization/filter/DistributedSerializableObjectConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.serialization.filter;
```

Comment:
@jchen21 noticed that there are two versions of DistributedSerializableObjectConfig in different packages. LocatorLauncher should be changed to use org.apache.geode.internal.DistributedSerializableObjectConfig instead of this one. Then this one should be deleted because it causes a split-package with classes in both geode-serialization and geode-core living in org.apache.geode.internal.serialization.filter (Java 9 does not allow split packages in different modules for the reason of the reliable configuration).

Reply:
Fixed